.github/workflows/test.yml

```diff
@@ -31,24 +31,14 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: ${{ inputs.GO_VERSION }}
-      - name: Test & Coverage
-        run: make cover
-      - uses: codecov/codecov-action@v3.1.4
+      - name: run unit test
+        run: make test-unit
+      - name: upload coverage
+        uses: codecov/codecov-action@v3.1.3
         with:
+          token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.txt
 
-  unit_test:
```

Comment:
sorry, can we actually keep this name? so update https://github.com/rollkit/rollkit/pull/968/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R25 to be `unit_test`?

Reply:
fixed